dev/firely/docker-compose.sqlite.yaml

```
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+
+  firely:
+    environment:
+      VONK_Repository: SQLite
+      VONK_SQLiteDbOptions:ConnectionString: "Data Source=./data/vonkdata.db"
+      VONK_SQLiteDbOptions:AutoUpdateDatabase: "true"
```

The primary SQLite DB should probably be on a docker volume, so it's not accidentally deleted (from the checkout)


```suggestion
      VONK_SQLiteDbOptions:AutoUpdateDatabase: "true"
    volumes:
      - db-data:/app/data
volumes:
  db-data: {}
```